### PR TITLE
TSK-2185: upgrade postgres to version 14.7

### PR DESCRIPTION
**Pull request by ryzheboka**
Edited documentation: 
https://taskana.atlassian.net/wiki/spaces/TAS/pages/698286081/Using+multiple+databases+for+taskana-core+unit+tests
https://taskana.atlassian.net/wiki/spaces/TAS/pages/1758593073/Supported+Environments
Sonarcloud: https://sonarcloud.io/summary/new_code?id=ryzheboka_taskana&branch=TSK-2185

Issue Link: #2185

---
Release Notes:

```

```
 
### For the submitter:
- [x] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Review by holgerhagen — CHANGES_REQUESTED** [review]
Please change the documentation as well. The supported postgres version is named there, isn't it?

File: common/taskana-common-test/src/main/java/pro/taskana/common/test/DockerContainerCreator.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public static Optional<JdbcDatabaseContainer<?>> createDockerContainer(DB db) {
                     "-c",
                     "localedef -i de_DE -c -f UTF-8 -A /usr/share/locale/locale.alias de_DE.UTF-8 "
                         + "&& export LANG=de_DE.UTF-8 "
-                        + "&& ./docker-entrypoint.sh postgres -c fsync=off")
+                        + "&& /usr/local/bin/docker-entrypoint.sh postgres -c fsync=off")
```
[the remark this answers is not in the document]
Comment (ryzheboka) [reply common/taskana-common-test/src/main/java/pro/taskana/common/test/DockerContainerCreator.java:55]: When using ./docker-entrypoint, there is the following error: 
./docker-entrypoint.sh: not found

When not using any command (thus not specifying any entrypoint), docker prints out the new location of the entrypoint (/usr/local/bin). [This tutorial](https://levelup.gitconnected.com/creating-and-filling-a-postgres-db-with-docker-compose-e1607f6f882f) also says: "In our Postgres container, we will find this bash script /usr/local/bin/docker-entrypoint.sh"

**Comment by ryzheboka** [comment]
Documentation only mentions postgres in an example here: https://taskana.atlassian.net/wiki/spaces/TAS/pages/698286081/Using+multiple+databases+for+taskana-core+unit+tests. I changed the version mentioned there. I searched for postgres in our documentation, but all other mentions of postgres seem not to reference the used version.

**Comment by mustaphazorgati** [comment]
> Documentation only mentions postgres in an example here: https://taskana.atlassian.net/wiki/spaces/TAS/pages/698286081/Using+multiple+databases+for+taskana-core+unit+tests. I changed the version mentioned there. I searched for postgres in our documentation, but all other mentions of postgres seem not to reference the used version.

Don't forget this: https://taskana.atlassian.net/wiki/spaces/TAS/pages/1758593073/Supported+Environments ;)

**Comment by ryzheboka** [comment]
Thanks @mustaphazorgati! I missed this page by searching "postgres" in the documentation :) It's edited now.

**Review by holgerhagen — APPROVED** [review]

**Pull request merged**